packages/mobile/src/qrcode/utils.ts

```diff
@@ -111,6 +115,11 @@ export function* handleBarcode(
     yield call(initialiseWalletConnect, barcode.data, WalletConnectPairingOrigin.Scan)
     return
   }
+  if (barcode.data.startsWith('celo://wallet/payment')) {
```

Comment:
Can we use a different deeplink? We already have`celo://wallet/pay?xxx` which are unfortunately matched using `startsWith` in https://github.com/valora-inc/wallet/blob/712593a970a36f222864022751ff66ac753f02cf/packages/mobile/src/app/saga.ts#L251
So they could be incorrectly handled if scanned via the device camera and passed as a deeplink.

Maybe `celo://wallet/merchantPay`?

Additionally, it would be nice to handle these links inside the existing `handleDeepLink` method too.

Reply:
While I 100% agree, we need to sync with first-dag to know what link we should change it too. meanwhile I added it link this to take priority for now.

![Screenshot 2022-03-08 at 16 38 19](https://user-images.githubusercontent.com/3650621/157271590-11fc2a63-2bd9-4b69-a46b-9013719804a5.png)

Reply:
I talked to our current partner about switching to `merchant-payment` but probably we should do it after celo connect